app/views/groups/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<p style="color: green"><%= notice %></p>
+
+<br/>
+
+<div id="groups">
+  <% @groups.each do |group| %>
+    <%= link_to group_path(group), class: "group-link" do %>
+      <%= render group %>
+    <% end %>
+  <% end %>
+</div>

```

Comment:
- [ ] Also for your "Categories" page:

|Template|Your design|
|---|---|
| ![image](https://github.com/joyapisi/budget-app/assets/77069011/b22b9d4a-3f5d-4685-b000-0effbc3715bd) | ![image](https://github.com/joyapisi/budget-app/assets/77069011/018d616c-242f-458a-884f-c91057b50a97) |

The button should be fixed to the bottom, each category card should fit the width of the screen and the navigation bar could be improved to match the design! 🤓 🔧